### ISSUE TITLE
No notification for private messages

### ISSUE THREAD

**Issue by rebeka-catalina** [friendica/friendica#6368]
### Expected behavior
Notification counter should appear and/or be increased in case of new private message.

### Actual behavior
It is only shown in the popup menu when hovering "username"-menu in vier.

### Steps to reproduce the problem
receive a private message

### Friendica version you encountered the problem
2019.01-rc 1292

see `example.com/friendica` on your Friendica node for the version information.

### Friendica source (git, zip)
git

### PHP version
7.2.10

### SQL version
15.1 Distrib 10.1.37-MariaDB

**Comment by MrPetovan** [comment]
This is a behavior I intentionally changed recently. The unread private message notification is still shown:
- In a number badge on the private message icon in the header menu
- In the `/notifications` page

I removed it from the notification dropdown because it was:
- A duplicate of the private message icon badge
- A "fake" notification, which means that it wasn't clearing the actual notification still displayed in `/notifications` when clicking on it.

**Comment by rebeka-catalina** [comment]
> The unread private message notification is still shown:
> In a number badge on the private message icon in the header menu

This may be true with frio. In vier I must open a menu to see, if There are new private messages - in other words: in vier I don't get notified about incoming messages, if I don't look for it.

**Comment by MrPetovan** [comment]
Oh, I thought there was an unread badge on the private messages icon in vier as well.

**Comment by rebeka-catalina** [comment]
> Oh, I thought there was an unread badge on the private messages icon in vier as well.

No. In vier we now have a problem, because it was always displayed
with the same counter for other notifications.

**Comment by MrPetovan** [comment]
Duly noted.

**Comment by rebeka-catalina** [comment]
Are you sure? 
It was not only the counter. There was also an entry in that popup of the notifications. Those entries also don't appear anymore. 

![pm](https://user-images.githubusercontent.com/7385838/50651227-032e2900-0f83-11e9-8e09-56d7862d32c4.jpg)

**Comment by MrPetovan** [comment]
Yes, like I said they weren't actual notification items, which prevented the real notification items (displayed in `/notifications`) to be cleared when being clicked on.

My goal was to add a badge to the private message icon in vier but I realize there is no such thing. There are two possible ways to reconcile behaviors:
- Add the actual private message notifications items to the dropdown and remove the unread message badge on frio.
- Add a private message icon with unread badge on vier.

What would you rather have?

**Comment by rebeka-catalina** [comment]
Well, I think we should perhaps ask others as well.

In my oppinion there's no need to make a difference in notifications between PMs and other things. I mean for it is ok there are notifications in only one place for everything, but I don't know if others share this oppinion or not :-)

**Comment by MrPetovan** [comment]
Fair enough, here's the poll: https://friendica.mrpetovan.com/redir/6?url=display/735a2029-145c-2e49-a41a-44e428538705

**Comment by hoergen** [comment]
Until nobody voted for changing this behavior you should revert your changes. So that it is like everybody is expecting that since a long time.

**Comment by MrPetovan** [comment]
This will be decided one way or another tonight EST, we usually don't revert except for multiple site-breaking changes.

**Comment by hoergen** [comment]
So you decided to change it and the community should decide if your change should be reverted ? **IF they read it?** 
Cmon , are you serious?

**Comment by MrPetovan** [comment]
The change has been submitted through the normal review process that's open for everyone to see and discuss, and 16 hours to fix a change on a develop branch from the moment it is reported isn't a bad time. However if you want more consistent behavior, we also have a stable branch for you to use.

I understand your frustration but this change has been made according to the process and won't be an issue anymore tomorrow.

**Comment by AlfredSK** [comment]
> Until nobody voted for changing this behavior you should revert your changes.

I understand your point. But since when do we vote on every code change?
As it was before @MrPetovan code change these notifications were behaving like a _bug_ in frio: You got a notification on a new private message - you clicked on the notification to read the message. And when you went to the /notifications page after that, there was still a notification about the new message. 
Sometimes I get like 20 messages a day. Clicking all these redundant notifications to get rid of them can really be a pain. :-) So, a solution to this existing issue is welcome. But it's not easy to find a perfect solution for everyone because our themes are quite different as it turns out now.

**Comment by hoergen** [comment]
So because the notification wasn't removed (frio bug) in the notification area, as a bugfix there is no notification anymore for priv messages? If I got it wrong please correct me.

@MrPetovan I'm not frustrated, I'm astonished that this Idea "If frio can't fix that, we remove that feature from the core" is passing and nobody thinks about a consequent logic that an event should create always a notification. 
I must admit, that I'm a big fan of standard behavior, because that is reliable. If you we start making exceptions here and there it will be more and more difficult to understand the behavior of friendica. 

Is there no way Frio can fix the bug?

**Comment by MrPetovan** [comment]
Maybe I didn't make myself clear enough.

Formerly, on frio, private message notifications:
1. Appeared in the private message icon badge
2. Appeared in the notification icon badge
3. Appeared in the notification dropdown
4. Appeared in the notification page (`/notifications`)

The problems was the notification display duplication and that interacting with any of the three notifications above didn't clear 4.

After the change (basically removing 2. and 3.), on frio, private message notifications:
1. Appear in the private message icon badge
2. Appear in the notification page

And interacting with the unread private message clears both 1. and 2.

Meanwhile, on vier, formerly private message notifications:
1. Appeared in the notification icon badge
2. Appeared in the notification dropdown
3. Appeared in the notification page

Since that, unbeknownst to me at the time, there isn't any private message icon with a notification badge, removing 1. and 2. with my change effectively hid unread private message notifications from the main vier interface.

My two possible ways of solving this problem on vier is either:
- to make vier more like frio and add a private message icon with a notification badge.
- to make frio more like vier and restore the private message notifications in the notification dropdown while removing the duplicate notification badge from the private message icon.

The reason both themes have to be matching on this feature one way or the other is that the underlying mechanism for notifications (badge numbers, notification dropdown list items) is the same no matter the theme you pick.
